client/src/document/hooks.ts

```diff
@@ -26,24 +26,23 @@ export function useCopyExamplesToClipboard(doc: Doc | undefined) {
 
     [...document.querySelectorAll("div.code-example pre")].forEach(
       (element) => {
-        const userMessage = document.createElement("span");
+        const wrapper = element.parentElement;
```

Comment:
Yeah, I'm not sure how a parentElement could be falsy since we are selecting only `<pre>` tags that are child elements of `<div class="code-example">` containers. I agree, just bail early since the failure cases are entirely clear at the moment.